core/pom.xml

```diff
@@ -142,62 +142,53 @@
     </dependency>
   </dependencies>
 
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-clean-plugin</artifactId>
-        <configuration>
-          <filesets>
-            <fileset>
-              <directory>${project.basedir}/src/main/java/com/sap/ai/sdk/core/client</directory>
-              <includes>
-                <include>**/*</include>
-              </includes>
-            </fileset>
-          </filesets>
-        </configuration>
-        <executions>
-          <execution>
-            <id>delete-aicore-generated-client</id>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
The delete-aicore-generated-client step is missing in the new profile.
Same for the orchestration client but it should skip, because generation is disabled.

Reply:
It's not needed, the deletion happens as part of the genereation, I enabled the flag. It only deletes packages now, not the full directory, so it doesn't delete the custom code